packages/ui/src/ui/utils/strawberryControllerApi.ts

```diff
@@ -110,6 +111,7 @@ type StrawberryApiType = <
     cluster: string,
     params: ApiItem['params'],
     props: {
+        type?: QueryEngine.CHYT | QueryEngine.SPYT;
```

Comment:
type?: 'chyt' | 'spyt'

Reply:
I don’t understand why you don’t like enum. Now it is done more centrally

Reply:
The enum contains unnecessary values that not supported by the api